modules/carousel/Carousel.tsx

```diff
@@ -83,6 +84,18 @@ export interface ICarouselProps {
    * default to true
    */
   animateScroll?: boolean;
+  /**
+   * if true it'll darken furthest items
+   * from the center
+   * Default: true
+   */
+  darkenFurthestItems?: boolean;
+  /**
+   * if true it'll try to center an item
+   * when scroll is untouched for 400ms
```

Comment:
The scroll eventListener is debounced by 300ms. Should this comment be changed to reflect what is happening or should the code be adjusted?